tests/Stubs/TestPage.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace SilverStripe\DocumentConverter\Tests\Stubs;
+
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\DocumentConverter\PageExtension;
+use Page;
+
+class TestPage extends Page implements TestOnly
+{
+    private static $extensions = [PageExtension::class];
+    private static $defaults = [
+        'Content' => '<h1>Default TestPage</h1><p>With pre-import content.</p>'
+    ];
```

Comment:
Add a `$table_name`, otherwise 4.0.0 will throw warnings for not having it (silly, I know)